workers/unity/Assets/Playground/Scripts/Cubes/TriggerColorChangeSystem.cs

```diff
@@ -37,22 +39,27 @@ protected override void OnCreateManager(int capacity)
 
         protected override void OnUpdate()
         {
-            if (World.GetExistingManager<TickSystem>().GlobalTick % 100 != 0)
+            if (Time.time < nextColorChange)
             {
                 return;
             }
 
-            var newColor = (Generated.Playground.Color) colorValues.GetValue(new Random().Next(colorValues.Length));
+            nextColorChange = Time.time + 2;
```

Comment:
Might be better to extract the 2 into a constant rather than having it as a magic number.